atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -165,9 +165,15 @@ public int socketQueryTimeoutMillis() {
         return 62 * 1000;
     }
 
+    /*
+        Only allow this many extra enqueued requests above the configured poolsize onto a given node in the pool.
+        Attempts to enqueue more than this will immediately fail and instead fail over to another node as coordinator.
+        A person needing to modify this may have not enough backpressure in their application,
+        or have too much parallelism.
+     */
     @Value.Default
-    public int cqlPoolTimeoutMillis() {
-        return 20 * 1000;
+    public int cqlPoolMaxQueueSize() {
```

Comment:
This change would not be safe to implement (would cause errors from people with the old one set), except no one uses this KVS / vis-a-vis this parameter yet so wouldn't have an existing value set.